libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -3604,6 +3604,70 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
     ],
   },
   // Light Integrations are at the end of the list until we figure out a way to fix the ticker/managerApp collisions
+  ethereum_as_evm_test_only: {
```

Comment:
I'm a bit concerned by this idea as it's the first time we actually break the 1 currency = 1 ledger currency relashionship. the coin model is not designed to be duplicated per network (afaik it's the first proposal doing this) and this can be a root cause of future obscure issues. for instance, the reasons we still need "findTokenByTicker" may be impacted ❓ 

To fix this, we should move toward the idea to have multiple backends among the same family. in the past, this was made possible, i think it still is: `libcore` and `js` was living at same time, and it was possible to make a switch between bridges, and we even had ways for user to switch the implementations. this system is still in place as we have `mock` vs `js`.

we may need to make `evm` & `ethereum` family one again & then it could internally change based on the `js:*` prefix of the account id (this is the way it was working for libcore)

I imagine we have such plan in mind for the future, in that case it would be good to be sure we mention it in the code, because this is going to add some tech debt.

Reply:
Those currencies are for test purpose only and hidden behind the "developer" mode since they're added as "testnets of" the original currency. I'm not sure to understand what kind of issues could arise from such "clone" currencies ? 🤔
Also tokens for "evm" currencies are only added as preload of a currency (so they're not as default in the @ledgerhq/cryptoassets lib), so *if* you had any issue with `findTokenByTicker` you'd have to be a dev to have it I guess ? (I can only find 1 `findTokenByTicker` usage in the whole codebase — the cli of Solana in this case — so I'm not sure when/how it's used, but "byTicker" to me would probably be used for counter values, where the Tokens would be identical then I guess ? Let me know if you have a specific case in mind.)

The plan is 100% to have only 1 family at the end, this PR is part of the process of adding every functionnalities of the "Ethereum" family to the EVM one, to be able to merge the 2, and have the "multiple backends among the same family" you're talking about 👍